### PR TITLE
Several minor changes to the doc:

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses [doc/#490](https://github.com/Avaiga/taipy-doc/issues/490).

- A bit more clear on on_user_content
- Web -> web when relevant
- ...

### CODE REVIEW

File: gui/src/components/Taipy/Chart.tsx [hunk 1]
```diff
@@ -417,7 +417,7 @@ const Chart = (props: ChartProp) => {
                 console.info(`Error while parsing Chart.plot_config\n${(e as Error).message || e}`);
             }
             if (typeof plconf !== "object" || plconf === null || Array.isArray(plconf)) {
-                console.info("Error Chart.plot_config is not a dictionnary");
+                console.info("Error Chart.plot_config is not a dictionary");
```
Comment (FredLL-Avaiga) [inline gui/src/components/Taipy/Chart.tsx:420]: oups

File: src/taipy/gui/_gui_cli.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def create_parser(cls):
             nargs="?",
             default="",
             const="",
-            help="The path to the WebApp to be used. The default is the webapp directory under gui in the Taipy GUI package directory.",
+            help="The path to the web app to be used. The default is the webapp directory under gui in the Taipy GUI package directory.",
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/_gui_cli.py:31]: not web application here ?

**Review by FredLL-Avaiga — APPROVED** [review]
🍟

File: src/taipy/gui/_gui_cli.py (hunk 2 above)
Comment (FabienLelaquais) [reply src/taipy/gui/_gui_cli.py:31]: Here is how I used those terms: web app indicates the Taipy GUI web app, the one in "webapp/".
A web application is what a programmer could build, using Taipy GUI.

**Pull request merged**